frontend/src/components/Evaluation/tabTitle.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { useHistory, useLocation } from "react-router-dom";
+import "./tabTitle.scss";
+
+const TabTitle = ({ link, children }) => {
+  const history = useHistory();
+  const pathname = useLocation().pathname;
+  return (
+    <li
+      className={link === pathname && "is-active"}
+      onClick={() => history.push(link)}
```

Comment:
why don't you use NavLink from react-router?

Reply:
I ended up using the `a` tag with `href` as the browser was complaining about the lack of `href`. And bulma needs an `a` tag for styling of tabs